rebasehelper/plugins.py

```diff
@@ -46,5 +63,175 @@ def load(cls, entrypoint):
                 continue
             else:
                 plugin.name = ep.name
+                # Some plugins require access to other plugins. Avoid cyclic
+                # imports by setting runner as an attribute.
+                plugin.runner = runner
             result[ep.name] = plugin
         return result
+
+
+class PluginRunner(object):
+    PLUGIN_ENTRYPOINTS = {
+        'build_tools': 'rebasehelper.build_tools',
+        'srpm_build_tools': 'rebasehelper.srpm_build_tools',
+        'checkers': 'rebasehelper.checkers',
+        'spec_hooks': 'rebasehelper.spec_hooks',
+        'build_log_hooks': 'rebasehelper.build_log_hooks',
+        'versioneers': 'rebasehelper.versioneers',
+        'output_tools': 'rebasehelper.output_tools',
+    }
```

Comment:
This mapping is completely unnecessary, you can use `pkg_resources.get_entry_map('rebasehelper')` to get a dictionary of all groups and their entrypoints. Then you could just strip `rebasehelper.` prefix and use that as an attribute name.